### DISCUSSION TITLE
How to get badge pro?

### DISCUSSION THREAD

**Discussion by AntyBD** [community/community#21002] [Other Feature Feedback, Questions, & Ideas]
how can i get pro badge?

**Comment by MrSarno** [comment]
Hi @AntyBD,

You get the Pro badge while maintaining [a Pro subscription](https://github.com/settings/billing/plans) for $4 / month.

**Comment by fxip** [comment]
I think if u are student u can apply it for free here https://education.github.com/pack 
or pay for it if u are not student

**Comment by fernandounger** [comment]
> How to get badge pro?

If you can prove to Github that you are a student, you can get badge github pro. You can apply for the student developer pack here: https://education.github.com/discount_requests/student_application

**Reply by fillahaufi** [reply]
![image](https://user-images.githubusercontent.com/63279983/186570618-6f6f0213-62ba-4774-8961-5830b6e971da.png)

I was approved on GitHub Education Pack, but my profile doesn't seem like changing to the pro version. Do you know why that happens? A little information would be very helpful. Thank you.

**Reply by zobaer53** [reply]
Add your student email account ez.
